.github/workflows/runtests.yml

```diff
@@ -111,7 +111,7 @@ jobs:
           python-version: 3.9
       - run: python -m pip install --upgrade pip
       - run: pip install mpmath
-      - run: TRAVIS_BUILD_NUMBER=true bin/test --force-colors --verbose --slow --timeout=595 --split=1/2
+      - run: TRAVIS_BUILD_NUMBER=true bin/test --force-colors --slow --timeout=595 --split=1/2
```

Comment:
I think we should keep the `--verbose` flag

Reply:
Sure, will keep it.
Thanks.